bundles/org.openhab.binding.kaleidescape/README.md

```diff
@@ -13,10 +13,10 @@ All movie player components including the original K-Player series, M Class Play
 It is important to choose the correct thing type to ensure the available channels are correct for the component being used.
 
 The supported thing types are:
-`player` Any KPlayer, M Class [M300, M500, M700] or Cinema One 1st Gen player
-`cinemaone` Cinema One (2nd Gen)
-`alto`
-`strato` Includes Strato, Strato S, Strato C or Strato V
+- `player` Any KPlayer, M Class [M300, M500, M700] or Cinema One 1st Gen player
```

Comment:
If this is changed to a list, there should be an empty line:
MD032/blanks-around-lists: Lists should be surrounded by blank linesmarkdownlint[MD032](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md032.md)

```suggestion

- `player` Any KPlayer, M Class [M300, M500, M700] or Cinema One 1st Gen player
```